docs/services/slam/cartographer/_index.md

```diff
@@ -1,20 +1,24 @@
 ---
-title: "Cartographer Integrated Library"
+title: "Cartographer Modular Resource"
 linkTitle: "Cartographer"
 weight: 70
 type: "docs"
-description: "Configure a SLAM service with the Cartographer library."
+description: "Configure a SLAM service with the Cartographer modular resource."
 tags: ["slam", "services"]
+icon: "/services/img/icons/slam.svg"
 aliases:
   - "/services/slam/run-slam-cartographer"
 # SMEs: Kat, Jeremy
 ---
 
 [The Cartographer Project](https://github.com/cartographer-project) performs dense SLAM using LIDAR data.
 
+To easily integrate Cartographer into the Viam ecosystem, Viam provides the [`viam-cartographer`](https://github.com/viamrobotics/viam-cartographer) module which wraps Cartographer as a [modular resource](/program/extend/modular-resources/) in the form of a custom SLAM service.
+This module provides the `cartographer-module` binary, which runs in addition to `viam-server`.
```

Comment:
```suggestion
```
I'm not sure this sentence is helpful if we are not clarifying where the two are running-- seems slightly confusing

Reply:
Gotcha. I really need to include mention of `cartographer-module` here, since this rest of this page uses this term over the others introduced in this paragraph. Otherwise there's nothing to connect that:

the module `viam-cartographer` installs the binary `cartographer-module`.

Reply:
Ah ok, great point. I was looking over this again and realized what could actually be confusing is referring to `viam-cartographer` as the module-- `cartographer-module` is the module name. The " `cartographer` library" (viam:slam:cartographer) " is the model of SLAM service provided by this module, hence it itself is the modular resource. If you make that clarification this will be perfect, agree more info is needed!